ui/src/components/credentials/IssuedTable.tsx

```diff
@@ -1,53 +1,302 @@
+    const parsedCredentialStatus = credentialStatusParser.safeParse(value);
+    if (parsedCredentialStatus.success) {
+      const params = new URLSearchParams(searchParams);
+
+      if (parsedCredentialStatus.data === "all") {
+        params.delete(STATUS_SEARCH_PARAM);
+      } else {
+        params.set(STATUS_SEARCH_PARAM, parsedCredentialStatus.data);
+      }
+
+      setSearchParams(params);
+    } else {
+      processZodError(parsedCredentialStatus.error).forEach((error) => void message.error(error));
+    }
+  };
+
+  useEffect(() => {
+    const { aborter } = makeRequestAbortable(fetchCredentials);
+
+    return aborter;
+  }, [fetchCredentials]);
+
   return (
     <TableCard
       defaultContents={
         <>
           <Avatar className="avatar-color-cyan" icon={<IconCreditCardRefresh />} size={48} />
 
           <Typography.Text strong>No credentials</Typography.Text>
+
           <Typography.Text type="secondary">
             Issued credentials will be listed here.
           </Typography.Text>
 
-          <Link to={generatePath(ROUTES.issueCredential.path)}>
-            <Button icon={<IconCreditCardPlus />} type="primary">
-              {ISSUE_CREDENTIAL}
-            </Button>
-          </Link>
+          {credentialStatus === "all" && (
+            <Link to={generatePath(ROUTES.issueCredential.path)}>
+              <Button icon={<IconCreditCardPlus />} type="primary">
+                {ISSUE_CREDENTIAL}
+              </Button>
+            </Link>
+          )}
         </>
       }
-      isLoading={false}
-      onSearch={() => null}
-      query={null}
+      isLoading={isAsyncTaskStarting(credentials)}
+      onSearch={onSearch}
+      query={queryParam}
       searchPlaceholder="Search credentials, attributes, identifiers..."
-      showDefaultContents={true}
+      showDefaultContents={showDefaultContent}
       table={
         <Table
-          dataSource={[]}
+          columns={tableColumns.map(({ title, ...column }) => ({
+            title: (
+              <Typography.Text type="secondary">
+                <>{title}</>
+              </Typography.Text>
+            ),
+            ...column,
+          }))}
+          dataSource={credentialsList}
+          locale={{
+            emptyText:
+              credentials.status === "failed" ? (
+                <ErrorResult error={credentials.error.message} />
+              ) : (
+                <NoResults searchQuery={queryParam} />
+              ),
+          }}
           pagination={false}
           rowKey="id"
           showSorterTooltip
           sortDirections={["ascend", "descend"]}
         />
       }
       title={
-        <Row justify="space-between">
-          <Space size="middle">
-            <Card.Meta title={ISSUED} />
+        <Row align="middle" justify="space-between">
+          <Space align="end" size="middle">
+            <Card.Meta title={ISSUE_CREDENTIAL} />
```

Comment:
This needs to read "Issued"
![Screenshot from 2023-04-12 16-40-24](https://user-images.githubusercontent.com/2691584/231493081-79bc1bd2-cc80-46c7-b49a-a03fc1aaad9e.png)